app/controllers/register/establishers/individual/PostCodeLookupController.scala

```diff
@@ -63,7 +63,7 @@ class PostCodeLookupController @Inject()(
               routes.PostCodeLookupController.onSubmit(mode, index, srn),
               routes.AddressController.onPageLoad(mode, index, srn),
               title = Message(title),
-              heading = Message(heading),
+              heading = Message(heading, details.fullName),
```

Comment:
We're doing an implicit conversion above  from `String` to `Message` more specifically a Literal

```
private val heading: Message = "messages__establisher_individual_address__heading"
```

resolving it implicity, so effectively:

```
heading = Message(resolve(heading)(messages), details.fullName),
```

and then wrapping it again as a `Message`, but this time as a `Resolvable`. We should instead do:

```suggestion
              heading = Message("messages__establisher_individual_address__heading", details.fullName),
```

and remove line 52